joss_paper/paper.md

```diff
@@ -0,0 +1,160 @@
+---
+title: 'alchemlyb: the simple alchemistry library'
+tags:
+  - Python
+  - alchemistry
+  - molecular dynamics
+  - free energy
+authors:
+  - name: Zhiyi Wu
+    orcid: 0000-0002-7615-7851
+    equal-contrib: true
+    affiliation: 1
+  - name: David L. Dotson
+    orcid: 0000-0001-5879-2942
+    equal-contrib: true
+    affiliation: "2, 3"
+  - name: David Mobley
```

Comment:
@DrDomenicoMarson can you please add yourself here (Name, Affiliation, OrcID if you have one)? The easiest way to do it is to edit the file on GitHub, which will create a PR against this PR. I can then merge your changes in.

If you can also edit the *Author Contributions* section below and add your initials under *contributed to new features* then we have everything from you.